stac/disasters/hurricane-harvey/hurricane-harvey-0831/20170831_195552_SS02/20170831_195552_SS02.json

```diff
@@ -1,18 +1,19 @@
 {
   "stac_version": "1.0.0",
   "stac_extensions": [
-    "https://stac-extensions.github.io/eo/v1.0.0/schema.json",
+    "https://stac-extensions.github.io/eo/v1.1.0/schema.json",
     "https://stac-extensions.github.io/view/v1.0.0/schema.json",
-    "https://stac-extensions.github.io/projection/v1.0.0/schema.json",
-    "https://stac-extensions.github.io/raster/v1.1.0/schema.json"
+    "https://stac-extensions.github.io/projection/v1.1.0/schema.json",
+    "https://stac-extensions.github.io/raster/v1.1.0/schema.json",
+    "https://planetlabs.github.io/stac-extension/v1.0.0-beta.2/schema.json"
   ],
   "type": "Feature",
   "id": "20170831_195552_SS02",
   "collection": "planet-disaster-data",
   "properties": {
     "datetime": "2017-08-31T19:55:53.513448Z",
     "gsd": 1.1,
-    "platform": "SS02",
+    "platform": "SSC2",
```

Comment:
Same question as in https://github.com/planetlabs/open-stac/pull/3#discussion_r1218061445